src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -2498,6 +2498,11 @@ namespace winrt::TerminalApp::implementation
         _RemoveTabViewItem(tabViewItem);
     }
 
+    TermControl TerminalPage::_InitControl(TerminalApp::TerminalSettings& settings, ITerminalConnection connection)
```

Comment:
```suggestion
    TermControl TerminalPage::_InitControl(const TerminalApp::TerminalSettings& settings, const ITerminalConnection& connection)
```

This will save a reference count!

Reply:
Make sure to fix the .h file too

Reply:
Fixed!